js/js_corporate/app.js

```diff
@@ -0,0 +1,133 @@
+
+
+$(function(){
+
+// Fixed header start
+    let header = $("#header");
+    let intro = $("#intro");
+    let introH = intro.innerHeight();
+    let scrollPos = $(window).scrollTop();
+
+    let nav = $("#nav");
+    let navToggle = $("#navToggle");
+
+    checkScroll(scrollPos, introH);
+
+    $(window).on("scroll resize", function(){
+        introH = intro.innerHeight();
+        scrollPos = $(this). scrollTop();
+
+        checkScroll(scrollPos, introH)
+    });
+
+    function checkScroll(){
+        if (scrollPos>introH) {
+            header.addClass("fixed");
+        } else {
+                header.removeClass("fixed");
+        }
+    };
+// Fixed header end
+
+
+
+// Smoth scroll start
+    $("[data-scroll]").on("click", function(event){
+        event.preventDefault();
+
+        let elementId = $(this).data('scroll');
+        let elementOffset = $(elementId).offset().top;
+
+        nav.removeClass("show");
+
+        $("html, body").animate({
+            scrollTop:elementOffset - 80
+        }, 700);
+    });
+// Smoth scroll end
+
+
+
+// Nav Toggle start
+    navToggle.on("click", function(event){
+        event.preventDefault();
+
+        nav.toggleClass("show");
+
+    })
+// Nav Toggle end
+
+
+});
+
+// Scroll for biigest element start
+let els = document.getElementsByClassName('element')
+document.addEventListener("scroll",elementPicker)
+document.addEventListener("DOMContentLoaded",elementPicker)
+
+function elementPicker(e){
+
+  for (let i = 0; i<els.length; i++) {
+    if ((els[i].getBoundingClientRect().top>30) & (els[i].getBoundingClientRect().top<300)) {
+        els[i].classList.add('bigusElementus')
+    } else els[i].classList.remove('bigusElementus')
+  }
+}
+// Scroll for biigest element end
+
+
+// Owl Carousel start
+var owl = $('.owl-carousel');
+owl.owlCarousel({
+    items:1,
+    loop:true,
+    margin:10,
+    nav: true,
+    mouseDrag: true,
+		touchDrag: true,
```

Comment:
what happened with tabs here?

Reply:
Mouse drag enabled.

https://owlcarousel2.github.io/OwlCarousel2/docs/api-options.html

Reply:
i meant the indentation (`touchDrag, pullDrag, freeDrag` seem to be much more indented than everything else)